packages/integrations/cloudflare/README.md

```diff
@@ -48,7 +48,13 @@ Cloudflare Pages has 2 different modes for deploying functions, `advanced` mode
 
 For most projects the adapter default of `advanced` will be sufficient; the `dist` folder will contain your compiled project. Switching to directory mode allows you to use [pages plugins](https://developers.cloudflare.com/pages/platform/functions/plugins/) such as [Sentry](https://developers.cloudflare.com/pages/platform/functions/plugins/sentry/) or write custom code to enable logging.
 
-In directory mode the adapter will compile the client side part of your app the same way, but moves the worker script into a `functions` folder in the project root. The adapter will only ever place a `[[path]].js` in that folder, allowing you to add additional plugins and pages middleware which can be checked into version control. Cloudflare documentation contains more information about [writing custom functions](https://developers.cloudflare.com/pages/platform/functions/).
+In directory mode, the adapter will compile the client side part of your app the same way by default, but moves the worker script into a `functions` folder in the project root. In this case, the adapter will only ever place a `[[path]].js` in that folder, allowing you to add additional plugins and pages middleware which can be checked into version control.
+
+Cloudflare documentation contains more information about [writing custom functions](https://developers.cloudflare.com/pages/platform/functions/).
+
+With the build configuration `split: true`, the adapter instead compiles a separate bundle for each page. This option requires some manual maintenance of the `functions` folder. Files emitted by Astro will overwrite existing `functions` files with identical names, so you must choose unique file names for each file you manually add. Additionally the adapter will never empty the `functions` folder of outdated files, so you must clean-up the folder manually when you remove pages.
+
+Additionally, this adapter does not support using [Cloudflare Pages Middleware](https://developers.cloudflare.com/pages/platform/functions/middleware/). Astro will bundle the [Astro Middleware](https://docs.astro.build/en/guides/middleware/) into each page.
```

Comment:
```suggestion
With the build configuration `split: true`, the adapter instead compiles a separate bundle for each page. This option requires some manual maintenance of the `functions` folder. Files emitted by Astro will overwrite existing `functions` files with identical names, so you must choose unique file names for each file you manually add. Additionally, the adapter will never empty the `functions` folder of outdated files, so you must clean up the folder manually when you remove pages.

Note that this adapter does not support using [Cloudflare Pages Middleware](https://developers.cloudflare.com/pages/platform/functions/middleware/). Astro will bundle the [Astro middleware](https://docs.astro.build/en/guides/middleware/) into each page.
```
Tiny nits to fix, including I just didn't like having "Additionally," twice in a row like that!  :sweat_smile: (Not wrong, just trying to go for some sentence variety. Either one could be changed!)

Also, the link to writing Cloudflare custom functions now feels a little out of place? Can we get away without it entirely?

Otherwise, happy to sign off on this form of the document!

Reply:
Makes sense :)

I think we can get away without the link, because the Middleware link also links to their docs. People writing custom functions, will probably know the Cloudflare Docs anyways.